contracts/Admin.cdc

```diff
@@ -70,7 +70,8 @@ pub contract Admin {
 				self.capability != nil: "Cannot create FIND, capability is not set"
 			}
 
-			self.capability!.borrow()!.setWallet(wallet)
+			let walletRef = self.capability!.borrow() ?? panic("Cannot borrow reference to receiver. ")
```

Comment:
Should these message include the address of the reciever we are trying to borrow from?